### ISSUE TITLE
not pulling the ds/jupyterhub:3.0.5 with the quickstart deployer

### ISSUE THREAD

**Issue by jarcher** [jupyter-on-openshift/jupyterhub-quickstart#19]
the DB is fine but the JH images are not pulling ds/jupyterhub:3.0.5 - perhaps it is just the 3.0.0. reference in the BC at jupyterhub-quickstart/build-configs/jupyterhub.json?

**Comment by GrahamDumpleton** [comment]
How you deploy it changed with 3.X. Did you check the README file? You would now usually run:

```
oc create -f https://raw.githubusercontent.com/jupyter-on-openshift/jupyterhub-quickstart/master/image-streams/jupyterhub.json
```

to load the image from ``quay.io``.

Anyway, that is the preferred approach. If you can't pull images and do need to build it yourself, the build configs are the correct way, but yes it seems I have been neglecting to update the version tag each time in those, so I will need to fix that.

**Comment by GrahamDumpleton** [comment]
Should be fixed now with 3.0.6 tag.

**Comment by jarcher** [comment]
I see that - I edited that for the 3.0.5 in the json file and it worked on another SAs minishift - thanks for the quick turnaround to your bits...
